### ISSUE TITLE
object isn't found

### ISSUE THREAD

**Issue by DeoLeung** [pgroonga/pgroonga#133]
```sql
-- above sql shows:
-- ERROR:  pgroonga: object isn't found: <Lexicon28443_0.index>
SELECT count(*) FROM "data-e9e927e5-e3a5-4b51-930a-8577712fc5bb";

-- this works
SELECT count(extra) FROM "data-e9e927e5-e3a5-4b51-930a-8577712fc5bb";

-- index ddl
CREATE INDEX "data-e9e927e5-e3a5-4b51-930a-8577712fc5bb_extra_idx" ON "data-e9e927e5-e3a5-4b51-930a-8577712fc5bb"(extra pgroonga_jsonb_ops_v2);

-- reindex works on our staging environment, but no effect in our production
REINDEX TABLE CONCURRENTLY "data-e9e927e5-e3a5-4b51-930a-8577712fc5bb";
```

I have checked that nothing special shows up in `pgroonga.log`, using `pgroonga==2.2.2`, is there any hint on how to troubleshoot  this problem?

**Comment by komainu8** [comment]
This problem is a known issue.
We have implemented a workaround for it in PGroonga 2.2.5.

Therefore, we need to upgrade to PGroonga 2.2.5 to solve this problem.

If PGroonga upgrade is difficult, we can solve it by using column name instead of `*` like `SELECT count(extra) FROM "data-e9e927e5-e3a5-4b51-930a-8577712fc5bb";`.

**Comment by DeoLeung** [comment]
got it, we will schedule an update later, thx
